package.json

```diff
@@ -10,12 +11,11 @@
     "Other"
   ],
   "activationEvents": [
-    "*"
+    "onStartupFinished"
```

Comment:
recommended instead of "*".
See: https://code.visualstudio.com/api/references/activation-events#onStartupFinished